migrations/2_deploy.js

```diff
@@ -46,9 +49,15 @@ module.exports = async function(deployer, network, accounts) {
   const currentBlockNumber = await web3.eth.getBlockNumber();
 
   await deployerApp.deploys([ZDAI, ZUSDC], txConfig);
-  console.log(`Deployed tokens: ZDAI [${ZDAI.address}] ZUSDC [${ZUSDC.address}] `);  
-  
+
+  console.log(`Deployed tokens: ZDAI [${ZDAI.address}] ZUSDC [${ZUSDC.address}] `);
+
+  // ATM Deployments
+  await deployerApp.deploy(ATMGovernance, txConfig); // TODO: add Gnosis multisig as signer/pauser (not the DAO for now)
```

Comment:
Take a look at the `feature/upgradable` branch for how to deploy an upgradable version of this. I can help if needed.

Reply:
This will be replaced by the ATMFactory